ui/pages/swaps/searchable-item-list/item-list/item-list.component.js

```diff
@@ -142,7 +142,7 @@ export default function ItemList({
                 ) : null}
               </div>
               {result.notImported && (
-                <Button type="confirm" onClick={onClick}>
+                <Button type="primary" onClick={onClick}>
                   {t('import')}
                 </Button>
               )}
```

Comment:
This is the only instance of the confirm button that I could find. @NiranjanaBinoy is there any reason that this type is used as opposed to primary type?

Reply:
I don't think so; I might have missed updating the type since it was already using the <Buttton/> component; my attention was on getting the rounded removed while updating that component.

Reply:
All good! Thanks for the update @NiranjanaBinoy 🙏